app/controllers/assignment_invitations_controller.rb

```diff
@@ -35,6 +49,19 @@ def check_user_not_previous_acceptee
     redirect_to success_assignment_invitation_path
   end
 
+  def classroom_config
```

Comment:
Nope, it only gets called during setup. So if it doesn't have a starter_code this shouldn't be called